list-package-updates

```diff
@@ -34,9 +34,8 @@ FILTER=pre_wrap
 SERIES=
 while [[ $1 = -* ]]; do
 case $1 in
-  --bare        ) FILTER=cat;      shift;;
-  -u|--upcoming ) SERIES=upcoming; shift;;
-             -* ) echo "Unsupported option: $1" >&2; usage ;;
```

Comment:
Thanks for catching this @edquist. I think this was due to a bad rebase when I changed remotes to point to github. I've removed the offending commit.